controllers/discovery/initDiscoveryUserData.js

```diff
@@ -12,6 +12,7 @@ const {MINIMUM_KARMA_SCORE} = require('../../helpers/constants');
  */
 const InitDiscoveryUserData = async (req, res) => {
   let {limit = 50, page = 0, allow_anon_dm} = req.query;
+  let {excluded_users = []} = req.body;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The extraction of `excluded_users` from the request body is done correctly. However, it would be better to ensure that `excluded_users` is an array before proceeding with the rest of the function. This can help prevent potential errors later in the code if `excluded_users` is not an array.
```diff
- let {excluded_users = []} = req.body;
+ let {excluded_users = []} = Array.isArray(req.body.excluded_users) ? req.body : [];
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->